packages/react-native/package.json

```diff
@@ -31,7 +31,8 @@
   "dependencies": {
     "@aws-amplify/ui": "6.1.0",
     "@aws-amplify/ui-react-core": "3.0.18",
-    "@aws-amplify/ui-react-core-notifications": "2.0.18"
+    "@aws-amplify/ui-react-core-notifications": "2.0.18",
+    "style-dictionary": "3.9.1"
```

Comment:
We've confirmed this is the version used by previous ui and this fixes the build issue when building an expo app?

Reply:
it is the version in the current yarn.lock file

Reply:
We have confirmed that this fixes the build issue when building an expo app